File: github/azawawi/perl6-msgpack/benchmark/test-msgpack.py
#!/usr/bin/env python

import msgpack

def test():
    SIZE     = 10000000;
    data     = [1] * SIZE
    packed   = msgpack.packb(data)
    unpacked = msgpack.unpackb(packed)

for i in range(1,10 + 1):
    test();

File: github/azawawi/perl6-msgpack/benchmark/test-msgpack.rb
#!/usr/bin/env ruby

require 'msgpack'

def test()
	size     = 10 * 1000 * 1000
	data     = [1] * size
    packed   = MessagePack.pack( data )
    unpacked = MessagePack.unpack( packed )
end

for i in 1..10 do
    test()
end

# vim: set ts=4 sw=4:
